gen/dcos-config.yaml

```diff
@@ -273,7 +273,7 @@ package:
                     [
                       {
                         "key": "output_file",
-                        "value" : "/var/lib/dcos/mesos/log/mesos-master.log"
+                        "value" : "{{ mesos_master_log_file }}"
```

Comment:
I was looking through the diagnostic tool code to see if there was a way to download a directory or multiple files based on a pattern.  We'll probably need to augment the diagnostics tool to include this ability because these logs are split into multiple files:
```
/var/lib/dcos/mesos/log/mesos-master.log
/var/lib/dcos/mesos/log/archive/mesos-master.log.1
/var/lib/dcos/mesos/log/archive/mesos-master.log.2.gz
/var/lib/dcos/mesos/log/archive/mesos-master.log.3.gz
/var/lib/dcos/mesos/log/archive/...
```

So in order to grab the full extent of logs, we'd need to download the `archive` folder as well.  Note: The exact number of log parts is not fixed.

Reply:
Note: Same applies for the agent logs.

Reply:
https://github.com/mesosphere/dcos-enterprise/pull/5138/files